### PR TITLE
feat: Symfony 7.0 support

### PR DESCRIPTION
**Pull request by grosmanal**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

This PR allows this bundle to be used with Symfony 7.0

### CODE REVIEW

**Comment by vincentchalamon** [comment]
Nice! Thanks @grosmanal.
Can you also update the .github/workflows/ci.yml to replace Symfony 6.3 by 6.4 (LTS), and add Symfony 7.0 in the matrix, please?

**Comment by grosmanal** [comment]
Converting to draft, because `behat/behat` and `friends-of-behat/symfony-extension` need to be updated to support Symfony 7.

**Comment by hanovruslan** [comment]
Hi, guys! Any progress here?

**Comment by grosmanal** [comment]
If the tests are green, I'll squash the commits. @vincentchalamon

File: composer.json [hunk 1]
```diff
@@ -18,38 +18,37 @@
...
         "api-platform/core": "^3.0",
         "behat/behat": "^3.1",
         "dg/bypass-finals": "^1.1",
         "doctrine/data-fixtures": "^1.2",
-        "doctrine/doctrine-bundle": "^1.7.2 || <2.7.1",
+        "doctrine/doctrine-bundle": "^2.11",
         "doctrine/orm": "^2.6.3",
-        "egulias/email-validator": "^2.1.10",
         "friends-of-behat/symfony-extension": "^2.0.11 || ^2.1.0",
         "jms/serializer-bundle": "^1.4 || ^2.3 || ^3.0 || ^4.0",
```
Comment (vincentchalamon) [inline composer.json:37]:
```suggestion
        "jms/serializer-bundle": "^1.4 || ^2.3 || ^3.0 || ^4.0 || ^5.0",
```

**Review by vincentchalamon — CHANGES_REQUESTED** [review]

**Review by vincentchalamon — APPROVED** [review]

**Comment by vincentchalamon** [comment]
Thanks @grosmanal! :rocket:

**Comment by vincentchalamon** [comment]
Published on v1.5.3

**Comment by hanovruslan** [comment]
Thanks a lot!

**Pull request merged**